Assets/Scripts/CommandPattern/ShootRaycast.cs

```diff
@@ -1,26 +0,0 @@
-﻿using UnityEngine;
-public class ShootRaycast
-{
-    private Camera rayCastCamera;
-    private Vector3 screenInputPosition;
-    public Vector2 RayCastPointMatrix { private set; get; }
-    public ShootRaycast(ICameraComponent camera, Vector3 clickPosition)
-    {
-        rayCastCamera = camera.GetSceneCamera();
-        screenInputPosition = clickPosition;
```

Comment:
Do you use it? Cause it seems that for detecting the hit you use Input.mousePosition below.